README.md

```diff
@@ -67,9 +67,11 @@ When developing a new component, always add a story for it. If you were creating
 
 ## Configuration
 
-You can configure the app by defining environment variables in `.env` files, the standard way to [configure Create React App](https://facebook.github.io/create-react-app/docs/adding-custom-environment-variables#adding-development-environment-variables-in-env).
+You can configure the app by defining environment variables in `.env` files, the standard way to [configure Create React App](https://facebook.github.io/create-react-app/docs/adding-custom-environment-variables#adding-development-environment-variables-in-env), but read on below because **there are some differences**.
 
-- How to add a new environment variable
+- How to override an environment variable for local development
+  - Define it in `.env.common-local`. **This differs** from how Create React App wants you to do it. If you put an environment variable in `.env.local`, it will get erased.
+- How to define a new environment variable
```

Comment:
```suggestion
- How to define a new environment variable?
```

Reply:
I think it's fine as a statement. If it were a question, the grammar would need to change, anyway. One example: "How do you define a new environment variable?" ... or ... "How would you define ...?"